composer.json

```diff
@@ -14,7 +14,8 @@
     ],
 
     "require": {
-        "bolt/filesystem": "^2.0",
+        "bolt/filesystem": "^2.1",
+        "contao/imagine-svg": "^0.1.2",
```

Comment:
What I mean is that in my observation, very few people are aware that `0.Y` counts as major in SemVer and behave as if it were a minor. I'm pretty sure I've seen core developers to it to.

For a require, `~` *might* make sense in most places, but project dependant, ad if I don't know for sure, I usually go for the loose option.

Third, we stay as-is, come back in a year and see which wat it went :wink: